<p>I am trying to install xgboost on my Mac for Python 3.4 but I'm getting the following error after "pip3 setup.py install":</p>  <pre><code>  File "&lt;string&gt;", line 20, in &lt;module&gt;    File "/private/var/folders/_x/rkkz7tjj42g9n8lqq5r0ry000000gn/T/pip-build-2dc6bwf7/xgboost/setup.py", line 28, in &lt;module&gt;      execfile(libpath_py, libpath, libpath)  NameError: name 'execfile' is not defined </code></pre>  <p>When running it with the -v option to get the verbose output the error looks like this:</p>  <pre><code>  Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/_x/rkkz7tjj42g9n8lqq5r0ry000000gn/T/pip-build-2dc6bwf7/xgboost Exception information: Traceback (most recent call last):   File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/basecommand.py", line 232, in main     status = self.run(options, args)   File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/commands/install.py", line 339, in run     requirement_set.prepare_files(finder)   File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/req/req_set.py", line 385, in prepare_files     req_to_install.run_egg_info()   File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/req/req_install.py", line 358, in run_egg_info     command_desc='python setup.py egg_info')   File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/pip/utils/__init__.py", line 749, in call_subprocess     % (command_desc, proc.returncode, cwd)) pip.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/_x/rkkz7tjj42g9n8lqq5r0ry000000gn/T/pip-build-2dc6bwf7/xgboost </code></pre>  <p>How can I solve this issue?</p>